cypress/integration/events_page.spec.ts

```diff
@@ -88,7 +88,19 @@ describe('/o/[orgId]/events', () => {
         //TODO: Verify that API request is done corrently.
     });
 
-    it('shows an undo sign-up button if user is signed up to an event', () => {
+    it.only('shows an undo sign-up button if user is signed up to an event', () => {
```

Comment:
The `it.only()` should not be included in the commit. In fact I thought we added a linting rule for this but I guess not yet.

Reply:
(Nope, the linting rule is part of #147)